SS14.Launcher/Views/Login/ExpiredLoginView.xaml

```diff
@@ -30,6 +30,11 @@
       <Button DockPanel.Dock="Left" HorizontalAlignment="Center"
               Content="Log out" Command="{Binding OnLogOutButtonPressed}" />
     </DockPanel>
+
+    <DockPanel DockPanel.Dock="Top" LastChildFill="False" MaxWidth="300" Margin="0 0 0 0">
+    <Button DockPanel.Dock="Left" HorizontalAlignment="Center"
+            Content="Forgot your password?" Command="{Binding ParentVM.SwitchToForgotPassword}" IsEnabled="{Binding !Busy}" />
+    </DockPanel>
```

Comment:
Remove unnecessary margin and fix indentation:
```suggestion
    <DockPanel DockPanel.Dock="Top" LastChildFill="False" MaxWidth="300">
      <Button DockPanel.Dock="Left" HorizontalAlignment="Center"
              Content="Forgot your password?" Command="{Binding ParentVM.SwitchToForgotPassword}" IsEnabled="{Binding !Busy}" />
    </DockPanel>
```
This whole file could use a cleanup, but it's fine for now